sunpy/map/maputils.py

```diff
@@ -1,7 +1,309 @@
 """
 This submodule provides utility functions to act on `sunpy.map.Map` instances.
 """
+from sunpy import map
+from sunpy.util import MetaDict
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from sunpy.sun import constants as con
+from sunpy.time import parse_time
+import numpy as np
+import random
+import matplotlib.pyplot as plt
+import astropy
+from sunpy.coordinates import frames
 
+__all__ = ['MapHelper']
 
-__all__ = []
 
+class MapHelper:
+    """
+    A helper class for knowing what is required in a `sunpy.map.Map` header.
+    A `sunpy.map.Map` header consists of a dictionary that contains meta information
+    about the map data. This meta is generally found in the FITS header of an image
+    file. This helper is defined to assist the user in finding out what meta data
+    keywords are accepted in a map header, and can produce a header MetaDict from a 
+    coordinate object, or through default meta that can be then passed into `Map` 
+    to create a Map object.
+
+    `map_helper.acceptable_meta_keywords()` prints the accepted meta that is used in 
+    creating a sunpy.map.Map
+
+    `map_helper.make_header(data)` will create a map header for the numpy.ndarray data, 
+    either with default meta keywords, or a SkyCoord or frame can be passed to extract
```

Comment:
```suggestion
    either with default meta keywords, or a `astropy.coordinates.SkyCoord` or frame can be passed to extract
```

I am not sure if that is right either. I forgot where SkyCoord lives.